### ISSUE TITLE
Solar capacity added to Norway?

### ISSUE THREAD

**Issue by Gazer75** [electricitymaps/electricitymaps-contrib#4649]
Seems @VIKTORVAV99  added some solar in #4496 using som basic NVE data?

This is not going to be accurate or even get any data reported to ENTSO-E as they are private installs on homes, barns and warehouses.
The system pretty much limits each customer to 100kW delivered to grid from these installs before being charged a production fee from the grid operator.
Even the big installs with several MWp on warehouses have this limit, but these usually only cover like max 25% of their total energy need from the panels anyway. So they would never actually deliver anything to the grid.

As of right now there is one 7MWp solar farm that have been given approval, which is Furuset solar farm along highway 3 between Rena and Koppang. Not sure if it is under construction already.
Another is Buer solar farm (1.1MWp) east of Sarpsborg, and a floating 1MWp test farm by Equinor on the northeast side of Frøya island in Trøndelag. Unknown status on these as well.

Apart from those there are two projects, Mæhlum (30-40MWp) and Øystadmarka (100-200 MWp), under consultation. Probably going to take a couple of years before decision is made on these.

So basically these current capacity values are not very useful to have in the map.

**Comment by VIKTORVAV99** [comment]
Hi @Gazer75, your correct that I added data solar data. And yes the data is incorrect but that's because it's outdated not anything else.

We use total installed grid connected capacity and that includes all kinds of types (residential, utility and everything between).

This is mostly because this data is the same for all countries so any comparisons between countries are fair.

As for the data reported to ENTSO-E that is up to the TSOs so in this case Statnett and should include residential solar production as well.

I hope this clears up things a little.

**Comment by VIKTORVAV99** [comment]
> > We use total installed grid connected capacity and that includes all kinds of types (residential, utility and everything between).
> 
> Which is why I point out that each install is technically max 100kW connected to grid regardless of how big the install itself is.

That might be true for residential solar but there is basically no residential installs at that size, it would be something like 250-300 solar panels.

And if the grid connection is limited to 100kw I doubt there is any private person willing to install much more than that anyway.

And any project bigger than that probably have utility connections and they are basically unlimited and only restricted by actual grid capacity. Like the big solar projects you provided as examples.

I really don't see the problem with including official production capacities here as we do for all other countries where we have the data.

**Comment by Gazer75** [comment]
By private I mean like warehouse owners like wholesale companies, the postal service terminals for Posten/Bring, stadiums, public buildings and so on. There are quite a few of these now that have several megawatts of solar on their roofs. They still can't deliver more than 100kW to the grid unless they pay an extra grid fee. This is still high enough that it would be a net loss in income unless you could consistently deliver over 100kW. But as we know solar doesn't do that.

The projects I mentioned will be connected directly to 22kV grid or higher and thus be like any other power plant/station. These need permission from NVE to be constructed. 
Private installs would feed back to the grid via the 230/400V connection to the nearby 22(11)kV substation. These do not need permission from NVE, but might need one from the municipality due to preserving the look of historic areas and such.


Bit OT, but in same vein is the last big gas power plant in regular operation at Melkøya near Hammerfest in NO4. It is normally isolated from the grid and produces power and heat for the LNG terminal only. I was told it is included in the reporting because it can support the grid via a 132kV connection if needed. As far as I know it has never done so since it was put into operation.

**Comment by VIKTORVAV99** [comment]
As for the capacity I think we will leave it as is since it's official numbers from NVE. What do you think @madsnedergaard?

As for the gas power plant at Melkøya this has been discussed before at #1430 as you know but I don't think there is anything to do here while it's connected to the grid. But this discussion should be continued in #1430.

**Comment by madsnedergaard** [comment]
Interesting discussion, thanks for bringing it up @Gazer75 :)
I see your point that it will most likely never hit the full capacity listed here, but since it's theoretically possible and we don't have a way of differentiating which installations are sending back to the grid and which are not, I'm unsure how we can do better.
 
While the current value is not ideal, I believe it's better to have **some** capacity listed than having **no** data at all.

If we can find a better official source for the capacity then that would of course be very welcome!

**Comment by Gazer75** [comment]
Hopefully solar will get more details in the future. I has been nothing but a footnote in the Norwegian energy mix so far with estimate of only 0.01% of total electricity production.
The current energy crisis and insane prices (1000-1500% of normal past 10 years) have created a boom for solar on private homes in the southern areas.

**Comment by madsnedergaard** [comment]
Let's hope the boom on solar will also bring increased transparency along the way! :)
I'll close this issue again since there's not much more we can do, but thanks again for bringing it up for discussion 🙏
